tests/api/local_buffer.cpp

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 The clvk authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "testcl.hpp"
+
+static bool check(cl_int* buf, size_t size, size_t limit) {
+    for (size_t i = 0; i < size; i++) {
+        if ((i >= limit && buf[i] != 0xffffffff) ||
+            (i < limit && (buf[i] != (limit - 1 - i)))) {
+            return false;
+        }
+    }
+    return true;
+}
+
+TEST_F(WithCommandQueue, LocalBuffer) {
```

Comment:
A name a bit more descriptive than `LocalBuffer` would have been nice but one can always use the git history to get the info.